lib/src/models/country_state.dart

```diff
@@ -1,21 +1,22 @@
 import 'package:json_annotation/json_annotation.dart';
 
-part 'state.g.dart';
+part 'country_state.g.dart';
 
 /// {@template states_struct}
 /// States in the database. Countries are divided into state.
 /// {@endtemplate}
 @JsonSerializable()
-class State {
+class CountryState {
```

Comment:
can't have this going in without it being a breaking change

Reply:
Well, i think it's worth it.  Is anyone serializing this?   I'm not sure how that would break anything that can't be quickly fixed.

Reply:
that's fine, but then it needs to be categorized as a breaking change and not only a code refactor